backend/src/models/employee.ts

```diff
@@ -0,0 +1,69 @@
+import { Sequelize, Model, DataTypes } from "sequelize";
+import sequelize from "src/modules/sequelize";
+
+export default class EmployeeModel extends Model {}
+
+EmployeeModel.init({
+    id: {
+        type: DataTypes.UUID,
+        defaultValue: DataTypes.UUIDV4,
+        primaryKey: true
+    },
+    first_name: {
+        type: DataTypes.STRING,
+        allowNull: false
+    },
+    last_name: {
+        type: DataTypes.STRING,
+        allowNull: false
+    },
+    gender: {
+        type: DataTypes.STRING,
+    },
+    dob: {
+        type: DataTypes.STRING,
+        allowNull: false
+    },
+    sin: {
```

Comment:
Should we stored Sin in our DB?

Reply:
I didn't find any regulation regarding to the storage of SIN, so I think it depends on the requirements of the stakeholder?